### DISCUSSION TITLE
[吐槽]我不是来看色图的啊

### DISCUSSION THREAD

**Discussion by WenSi-tofu** [share121/inter-knot#2949] [其他]
布什戈门，我只是来接点委托赚生活费顺带看绳斤们发癫的
色图什么的，我还不希望这个网站变得不正经啊
绳网团队管管吧色图佬吧！！！

**Comment by NiuBoss123** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by user51534** [comment]
我也不理解啊，我想找点内容，这种色图谁会看啊，有那需要的直接去看更色的就是了，不行单独开个色图板块或者帖子，要去找这种擦边的自己去翻<img src="https://img-static.mihoyo.com/communityweb/upload/9249bd36125b953ddd445b19151427d6.png" alt="安比 疑惑" width="75" >

**Comment by cpu644** [comment]
没人是来这看这个的。已经反馈了，明天管理起来删

**Comment by nninng** [comment]
其实就是他一个有病的在反复发，点举报就行了，希望他早日封号

**Comment by NiuBoss123** [comment]
虽然不反感发那些图，但我反感他一个帖就塞一张图，搞得他发了一堆帖子，氵
